lib/senkyoshi/models/questions/matching.rb

```diff
@@ -9,6 +9,16 @@ def initialize
       @distractors = []
     end
 
+    def strip_select_html(text)
+      text.gsub(/<\/?p[^>]*>/i, "").          # <p> tags
```

Comment:
I was curious so I did some (unscientific) testing and found that combining everything into one `gsub` looks to be 3 to 4 times faster.